app/src/main/java/org/sopt/sample/LoginActivity.kt

```diff
@@ -0,0 +1,84 @@
+package org.sopt.sample
+
+import android.app.Activity
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import androidx.activity.result.contract.ActivityResultContracts
+import com.google.android.material.snackbar.Snackbar
+import org.sopt.sample.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding : ActivityLoginBinding
+    private var userInfo = arrayOf("", "", "", "")
+
+    // 회원가입 페이지에서 입력한 정보 가져오기
+    private val resultLauncher = registerForActivityResult(
+        ActivityResultContracts.StartActivityForResult()
+    ) {result ->
+        if (result.resultCode == Activity.RESULT_OK) {
+            Snackbar.make(binding.root, "회원가입이 완료되었습니다!", Snackbar.LENGTH_SHORT).show()
+            val info : Intent? = result.data
+
+            val userName = info?.getStringExtra("name").toString()
+            val userId = info?.getStringExtra("id").toString()
+            val userPw = info?.getStringExtra("pw").toString()
+            val userMbti = info?.getStringExtra("mbti").toString().uppercase()
+
+            userInfo = arrayOf(userId, userPw, userName, userMbti)
+        }
+    }
+
+    // 아이디 비밀번호 입력란이 비었는지 판단
+    private fun isEmpty(inputId : String, inputPw : String) : Boolean {
+        return inputId == "" && inputPw == ""
+    }
```

Comment:
```suggestion
    private fun isEmpty(inputId : String, inputPw : String) : Boolean  = (inputId.isEmpty() && inputPw.isEmpty())
    
```
한 줄짜리 return 함수는 `=` 을 통해서 코드를 더 간단히 만들 수 있습니다~!

또, String 내장 함수 중에 `.isEmpty()` 라는 함수가 있어서 활용하면 좋을 것 같습니다!

Reply:
오... `isEmpty()` 내장함수가 있군요... 그럼 아예 이 함수를 없애도 무방할 것 같네요🤔
그 밖에 다른 한 줄짜리 함수에는 `=`을 사용해보겠습니다!